src/org/zaproxy/zap/extension/pscanrulesAlpha/ServerHeaderInfoLeakScanner.java

```diff
@@ -99,19 +98,25 @@ public int getPluginId() {
 	
 	@Override
 	public String getName(){
-		return Constant.messages.getString(MESSAGE_PREFIX + "name");
+		return Constant.messages.getString("pscanalpha.serverheader.scanner.name");

```

Comment:
I think this might break something in the API baseline example, however, using the plugin or alert names in exclusion/handling rules is probably a mistake anyway since they're i18n?